packages/mobile/src/escrow/actions.ts

```diff
@@ -88,3 +105,35 @@ export const resendPayment = (paymentId: string): ResendPaymentAction => ({
   type: Actions.RESEND_PAYMENT,
   paymentId,
 })
+
+export const setReclaimTransactionFee = (suggestedFee: string): SetReclaimTransactionFeeAction => ({
+  type: Actions.SET_RECLAIM_TRANSACTION_FEE,
+  suggestedFee,
+})
+
+export const fetchReclaimSuggestedFee = (paymentID: string) => async (
+  dispatch: DispatchType,
+  getState: GetStateType
+) => {
+  try {
+    Logger.debug(`${TAG}/fetchSuggestedFee`, `Fetching reclaim transaction fee`)
+    const escrow = await getEscrowContract(web3)
+    const stableToken = await getStableTokenContract(web3)
+    const account = currentAccountSelector(getState())
+
+    const mockReclaimTx = await escrow.methods.revoke(paymentID)
+
+    const mockTxParams: any = { from: account, gasCurrency: stableToken._address }
+    const gas: BigNumber = new BigNumber(await mockReclaimTx.estimateGas(mockTxParams))
```

Comment:
Nit: No need for the explicit setting of type (`: BigNumber`) in the next 3 lines
In general, the only time we should have to do that is for the result of a `yield` call. Hopefully even that case will be fixed soon :)